app/assets/stylesheets/spotlight/_bootstrap_overrides.scss

```diff
@@ -80,21 +80,22 @@ h1,h2,h3,h4,h5,h6 {
   }
 }
 
-.tab-pane {
```

Comment:
I _believe_ in the cases of the `<fieldset>`, `<legend>`, etc. we want the styles we've applied to be present regardless of whether it is in a tabbed display or not.

Reply:
Makes sense to me. The After definitely looks like what we want.